src/components/Charts/ChartZones.tsx

```diff
@@ -184,12 +186,14 @@ const ChartZoneAutosize = ({
   zones,
   capY,
   getTooltipText,
+  getPointText = (valueY: number) => formatPercent(valueY, 1),
```

Comment:
why do you set the default here?

Reply:
maybe we can add a comment i think this sets the floats to show

Reply:
is not really that useful, removing it as default and moving it to its corresponding chart